src/reducers/ui.js

```diff
@@ -187,6 +201,7 @@ export const sGetUiActiveModalDialog = (state) =>
     sGetUi(state).activeModalDialog
 export const sGetUiParentGraphMap = (state) => sGetUi(state).parentGraphMap
 export const sGetUiConditions = (state) => sGetUi(state).conditions || {}
+export const sGetUiRepetition = (state) => sGetUi(state).repetition
```

Comment:
```suggestion
export const sGetUiRepetition = (state) => sGetUi(state).repetition || {}
```
return empty object to prevent errors being thrown by the consumer if no repetition is found?

Reply:
Should not be needed when EMPTY_UI and DEFAULT_UI has `repetitionByDimension: {}`